src/apptesting/invokeTests.ts

```diff
@@ -0,0 +1,48 @@
+import { Client } from "../apiv2";
+import { appTestingOrigin } from "../api";
+import { InvokeTestCasesRequest, TestCaseInvocation, TestInvocation } from "./types";
+import * as operationPoller from "../operation-poller";
+import { FirebaseError } from "../error";
+
+const apiClient = new Client({ urlPrefix: appTestingOrigin(), apiVersion: "v1alpha" });
+
+export async function invokeTests(appId: string, startUri: string, testDefs: TestCaseInvocation[]) {
+  const appResource = `projects/${appId.split(":")[1]}/apps/${appId}`;
+  try {
+    const invocationResponse = await apiClient.post<
+      InvokeTestCasesRequest,
+      operationPoller.LongRunningOperation<TestInvocation>
+    >(`${appResource}/testInvocations:invokeTestCases`, buildInvokeTestCasesRequest(testDefs));
+    return invocationResponse.body;
+  } catch (err: unknown) {
+    throw new FirebaseError("Test invocation failed");
+  }
+}
+
+function buildInvokeTestCasesRequest(
+  testCaseInvocations: TestCaseInvocation[],
+): InvokeTestCasesRequest {
+  return {
+    resource: {
+      testInvocation: {},
+      testCaseInvocations,
+    },
+  };
+}
+
+export async function pollInvocationStatus(
+  operationName: string,
+  onPoll: (invocation: operationPoller.OperationResult<TestInvocation>) => void,
```

Comment:
could this periodically output the status (based on the metadata, i.e. running/succeeded/failed tests)